### PR TITLE
Move the utils function for the demos to be under the cluster class.

### PR DESCRIPTION
**Pull request by kfirtoledo**
Move the utils function for the demos to be under the cluster class,
for reusability, and supports the same demo in the kind and cloud environment.

### CODE REVIEW

**Review by elevran — CHANGES_REQUESTED** [review]
- there seems to be some missing newlines at end of files
- how does this relate to `iperf` changes in #170?

File: demos/bookinfo/kind/test.py [hunk 1]
```diff
@@ -23,133 +23,30 @@
...
-   runcmd(f'kubectl exec -i {gwctl3Pod} -- gwctl create policy --type access --policyFile {policyFile}')
-
-   print(f"Proctpage1 url: http://{gw1Ip}:30001/productpage")
-   print(f"Proctpage2 url: http://{gw1Ip}:30002/productpage")
+   gw1           = cluster(name="peer1")
+   gw2           = cluster(name="peer2")
+   gw3           = cluster(name="peer3")
 
+   bookInfoDemo(gw1, gw2, gw3, testOutputFolder, args["logLevel"], args["dataplane"])
+   print(f"Proctpage1 url: http://{gw1.ip}:30001/productpage")
```
Comment (praveingk) [inline demos/bookinfo/kind/test.py:50]: Typo productpage1

**Review by praveingk — CHANGES_REQUESTED** [review]

File: demos/bookinfo/kind/test.py (hunk 1 above)
Comment (kfirtoledo) [reply demos/bookinfo/kind/test.py:50]: Done

**Comment by kfirtoledo** [comment]
@elevran,  The main purpose of this change is that the same test (most of the common parts) will be run in kind and the cloud so that it will be easy to develop the demo in kind and then check it on the cloud.

**Comment by kfirtoledo** [comment]
@praveingk, @elevran PTAL

**Review by praveingk — APPROVED** [review]

**Pull request merged**